src/common/components/share/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from 'react';
+import Clipboard from 'clipboard';
+
+import styles from './styles.css';
+
+export class CopyToClipboard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isSupported: Clipboard.isSupported()
+    };
+  }
+
+  componentDidMount() {
+    if (this.state.isSupported) {
+      this.initClipboard();
+    }
+  }
+
+  componentWillUnmount() {
+    this.clipboard && this.clipboard.destroy();
+  }
+
+  initClipboard() {
+    this.clipboard = new Clipboard(this.copyBtn);
```

Comment:
Only add the eventListener to a single component instance, so multiple instances of this component can live with each other.